lib/relationships.js

```diff
@@ -98,6 +93,36 @@ function relationships (id, data, model) {
 
         updateRelation(modelTo, idToFind, where)
       })
+
+      if (serverRelation.modelThrough) {
+        var modelThrough = serverRelation.modelThrough
+        var key = keyByModel(modelThrough, modelTo)
+        var data = {}
+        data[fkName] = id
+
+        var payloadIds = _.map(relationship.data, function (item) {
+          return item.id
+        })
+
+        modelThrough.find({where: data}, function (err, instances) {
```

Comment:
Just a minor performance suggestion: it looks like you're only using one field from this query. Since this query is meant to operate on data instead of return it, you might add a `fields` filter to the `find` query to only pull out the keys you're using.

```javascript
modelThrough.find({where: data, fields: key})
```

I haven't profiled this, and the `modelThrough` will likely be pretty slim anyways, but it might have a minor performance benefit with larger datasets, since you're only querying and serializing the data you actually need.